package.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "zk-kit",
     "description": "A monorepo of reusable JS libraries for zero-knowledge technologies.",
+    "version": "0.1.1",
```

Comment:
This package should not have any version, since it's the monorepo root `package.json` file and it's private. We are not using it for versioning atm. I'll update the `incremental-merkle-tree.sol` package version (i.e. `packages/incremental-merkle-tree.sol/package.json`) as soon as your PR is accepted.

Reply:
removed